dask/bag/core.py

```diff
@@ -950,7 +950,7 @@ def distinct(self, key=None):
         return self.reduction(func, agg, out_type=Bag, name="distinct")
 
     def reduction(
-        self, perpartition, aggregate, split_every=None, out_type=Item, name=None
+        self, perpartition, aggregate, split_every=None, out_type=Item, name=None,
```

Comment:
I don't think you mean to change this file.

Reply:
Done on latest commit, thanks!